docs: update metabase store UI paths (#75136)
metabase/metabase: docs/cloud/accounts-and-billing.md

Managing billing and accounts in the Metabase Store.

## Upgrade to Pro (or downgrade to Starter)

To switch your plan, use the following steps:

1. Log in to your Metabase [Store account](https://store.metabase.com).
2. In the **Instances** section, select the instance you want to change.
3. Click **Manage plan**.
4. Select the plan you want.
5. Click **Change plan**.

## Add managers

To invite users to manage your Metabase Store account, use the following steps:

1. Log in to your Metabase [Store account](https://store.metabase.com).
2. Navigate to **Settings**.
3. Find the **Managers** section.
4. In the **Invite managers** field, enter the email address of each user you want to invite.
5. Metabase sends them an email prompting them to create a Metabase Store account.

> This action adds each invited user as a manager of your Metabase Store account only, not as a user of your Metabase instance.

## Add contacts to receive billing or technical notifications

> To add contacts who receive billing or technical notifications, use the following steps:

Reviewer: ```suggestion
To add contacts who receive billing or technical notifications:
```

1. Log in to your Metabase [Store account](https://store.metabase.com).
2. Navigate to **Settings**.
3. Find the **Contacts** section.
4. In the **Add contacts** field, enter the email address of each contact you want to add.

> 5. In the **Email type** drop-down, choose between **All**, **Billing**, and **Technical**.

Reviewer: ```suggestion
5. In the **Email type** dropdown, choose between **All**, **Billing**, and **Technical**.
```

Reviewer: Do they have to do anything else? Any button to hit?

6. Metabase sends each contact an email notifying them that they've been added to your contact list.

## Cancel a subscription or trial

To cancel a Starter or Pro plan, use the following steps:

1. Log in to your Metabase [Store account](https://store.metabase.com).
2. Navigate to **Instances**.
3. In the instance you want to cancel, click **Settings**.
4. Click **Cancel subscription and delete instance**. If you have an Enterprise plan, [contact support](https://www.metabase.com/help-premium).

> 5. Optionally, enter a cancellation reason and click **Share feedback**, or click **Skip**.

Reviewer: ```suggestion
5. Optional: let us know why you're cancelling so we can improve Metabase. Click **Share feedback**, or click **Skip**.
```
